### DISCUSSION TITLE
Bedrock mode and Structures

### DISCUSSION THREAD

**Discussion by Goretofame** [Zodsmar/SeedSearcherStandaloneTool#113] [Q&A] [answered]
Since in BedrockMode it is not possible to look for structures, if I look for them in normal mode is there a possibility that that seed matches or is it impossible?
Will it be possible to find a certain amount of structures in bedrock in the future?
Congratulations for the project, it is the only one I know for bedrock

**Comment by Zodsmar** [comment, marked as answer]
There are sometimes similarities. But it would need to be tested in bedrock or at least in a map viewer. But its a hit or miss

**Reply by Goretofame** [reply]
Use me for testing, i have a good pc to try everything you need. Let me help please

**Reply by Zodsmar** [reply]
So to come back to this question. With the changes to seeds in 1.18, bedrock mode wont exist anymore as seeds will be the same on both bedrock and java. So this problem shouldn't exist anymore as you will be able to search everything.

**Reply by Goretofame** [reply]
but we have not the same amount of seeds right? java has a lot more quantity for search by i know. The only change for bedrock is that we can go to negative seeds number. If im not wrong, is a easy fix to limit the seed quantity search for bedrock no? thanks! im always checking for new sassa versions :D a hug!
